pkgs/servers/fishnet/default.nix

```diff
@@ -33,5 +33,6 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://github.com/niklasf/fishnet";
     license = licenses.gpl3Plus;
     maintainers = with maintainers; [ tu-maurice ];
+    broken = !stdenv.isLinux || !(stdenv.isx86_64 || stdenv.isAarch64);
```

Comment:
Can we set platforms to x86_64-linux instead?

Reply:
Well, you told me earlier to mark it broken, but we sure can also just set the platforms instead.